apps/src/templates/sharing.html.ejs

```diff
@@ -49,9 +49,6 @@
       <button id="save-to-project-gallery-button">
         <%= msg.addToProjects() %>
       </button>
-      <button id="publish-to-project-gallery-button">
```

Comment:
@thomasoniii @bencodeorg - I removed this publish button since you both approved so re-requested your review.
I also updated a couple UI tests accordingly.